src/Category/Functor.agda

```diff
@@ -31,9 +31,9 @@ record RawFunctor {ℓ} (F : Set ℓ → Set ℓ) : Set (suc ℓ) where
 -- A functor morphism from F₁ to F₂ is an operation op such that
 -- op (F₁ f x) ≡ F₂ f (op x)
 
-record Morphism {ℓ} {F₁ F₂ : Set ℓ → Set ℓ}
+record Morphism {ℓ ℓ′ ℓ″} {F₁ : Set ℓ → Set ℓ′} {F₂ : Set ℓ → Set ℓ″}
                 (fun₁ : RawFunctor F₁)
-                (fun₂ : RawFunctor F₂) : Set (suc ℓ) where
+                (fun₂ : RawFunctor F₂) : Set ((suc ℓ) ⊔ (ℓ′ ⊔ ℓ″)) where
```

Comment:
Don't think you need any of those brackets for the levels.

Reply:
Indeed - fixing.